internal/gojs/fs.go

```diff
@@ -143,11 +143,22 @@ func (jsfsLstat) invoke(ctx context.Context, mod api.Module, args ...interface{}
 	path := args[0].(string)
 	callback := args[1].(funcWrapper)
 
-	lstat, err := syscallStat(mod, path) // TODO switch to lstat syscall
+	lstat, err := syscallLstat(mod, path)
```

Comment:
lstat is needed on GOOS=js because reading a directory involves fetching the dir names then an `lstat` per entry. This can't use `stat` because `stat` follows links.